lib/segment/src/index/field_index/tests/histogram_tests.rs

```diff
@@ -249,6 +251,15 @@ fn test_build_histogram() {
         .collect_vec();
 
     let (histogram, points_index) = build_histogram(max_bucket_size, precision, points);
+
+    // test persistence
+    let dir = tempfile::Builder::new()
+        .prefix("histogram_dir")
+        .tempdir()
+        .unwrap();
+    histogram.save(&dir).unwrap();
+    let histogram = Histogram::<f64>::load(&dir).unwrap();
+
```

Comment:
I suggest to not to mix things under the test. Instead, split into two parts:
In the first, run the old test without saving/loding.
In the second, do load/save and compare histograms using `assert_eq!()`.

Reply:
fixed. I reverted this change and added a separate unit test for test save-load